contracts/StakeManager.sol

```diff
@@ -66,7 +66,7 @@ contract StakeManager is Ownable {
         _;
     }
 
-    modifier onlyInitialized(address account) {
+    modifier onlyAccountInitialized(address account) {
```

Comment:
I've changed this one as `onlyInitialized` is commonly used in OZ `Initializable` contracts